### PR TITLE
[BETA] Swift 1.2 compatiblity

### LINKED ISSUE #9: Implicit returns in `layout` blocks
There seems to be an implicit return in layout blocks containing only a single statement.

```swift
layout(view, superview) { view, superview in
    view.top == superview.top + 100
}
```

throws over the compiler whereas

```
layout(view, superview) { view, superview in
    view.top == superview.top + 100
    return
}
```

doesn't. I guess the compiler treats the first block as `(LayoutProxy, LayoutProxy) -> NSLayoutConstraint`, in which case overloading `layout` should do the trick.

### PR DESCRIPTION
**Pull request by robb**
This branch is not intended to be merged until Swift 1.2 leaves beta

Fixes #9

### CODE REVIEW

File: CartographyTests/OperatorTests.swift [hunk 1]
```diff
@@ -44,7 +44,7 @@ class OperatorTests: XCTestCase {
         var constraint: NSLayoutConstraint!
 
         layout(view) { view in
-            constraint = view.width == (view.superview!.width + 100) * 2 + 5
+            constraint = view.width == (view.superview!.width + 100) * 2 + 5; return
```
Comment (robb) [inline CartographyTests/OperatorTests.swift]: Unfortunately, these give me an _expression too complex_ error without the `return` statement.

File: Cartography/ConstraintGroup.swift [hunk 2]
```diff
@@ -15,7 +15,7 @@ public class ConstraintGroup {
     @availability(iOS, introduced=8.0)
     public var active: Bool {
         get {
-            return constraints.map({ $0.layoutConstraint.active }).reduce(true, (&))
+            return constraints.map({ $0.layoutConstraint.active }).reduce(true) { $0 && $1 }
```
Comment (robb) [inline Cartography/ConstraintGroup.swift]: Not sure if this is intentional

**Comment by devxoul** [comment]
Travis CI doesn't support new beta of Xcode yet?

**Comment by robb** [comment]
Most likely not

**Comment by jamescmartinez** [comment]
Hey @robb, Swift 1.2 is out of beta. Let me know if I can help in any way.

**Pull request merged**